docs/src/tutorials/conic/simple_examples.jl

```diff
@@ -416,7 +427,38 @@ function example_robust_uncertainty_sets()
         sqrt((1 - ɛ) / ɛ) *
         sqrt(c' * (Σhat + Γ2(𝛿 / 2, N) * LinearAlgebra.I) * c)
     Test.@test objective_value(model) ≈ exact atol = 1e-2
-    return
+    return model
+end
+
+sdp_model_dict[:robust_uncertainty_sets] = example_robust_uncertainty_sets();
+
+# ## Comparison to dual formulation
+
+# It may be more efficient to solve the dual of a conic program rather than
+# the original formulation.
+# We can compare the numbers of variables and constraints in the primal formulations
+# given here and in the dual formulation provided by the [Dualization.jl](@ref) package:
+
+println("Model Name \t   #PrimalVars #Cons #DualVars #DualCons")
+println("------------------------------------------------------------")
+for (sym, model) in sdp_model_dict
+    ## Skip models with constraints types not yet supported:
+    if sym in [:correlation_problem, :robust_uncertainty_sets]
```

Comment:
What's not supported exactly ?

Reply:
For `correlation_problem`:
```
ERROR: Constraints of the Function MathOptInterface.VectorAffineFunction{Float64} in the set MathOptInterface.PositiveSemidefiniteConeSquare are not yet implemented.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] supported_constraints(con_types::Vector{Tuple{Type, Type}})
   @ Dualization ~/.julia/packages/Dualization/Xb4KB/src/supported.jl:15
```
For `robust_uncertainty_sets`:
```
ERROR: Constraints of the Function MathOptInterface.ScalarAffineFunction{Float64} in the set MathOptInterface.Interval{Float64} are not yet implemented.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] supported_constraints(con_types::Vector{Tuple{Type, Type}})
   @ Dualization ~/.julia/packages/Dualization/Xb4KB/src/supported.jl:15
```